skypelog.py

```diff
@@ -567,7 +650,7 @@ def usage():
 
 Option:
   -h, --help                Show this help message
-  -j, --json={compact,full} Save history for each user in *.js file (unsorted)
+  -j, --json={compact,full,calls} Save history for each user in *.js file (unsorted)
```

Comment:
This is slightly hamfisted but seemed like the most straightforward thing to do without changing the option parsing in a more substantial way as currently the action selected is entirely dependent on the output mode.